server/driver_tidb.go

```diff
@@ -196,11 +196,10 @@ func (qd *TiDBDriver) OpenCtx(connID uint64, capability uint32, collation uint8,
 	}
 	se.SetClientCapability(capability)
 	se.SetConnectionID(connID)
-	tc := &TiDBContext{
-		Session:   se,
-		currentDB: dbname,
-		stmts:     make(map[int]*TiDBStatement),
-	}
+	tc := new(TiDBContext)
```

Comment:
Please use the style on left side.

Reply:
Done.
I don't meant to change here, just to get the exact line to figure out how it races.